### ISSUE TITLE
Screen reader: when clicking on OPDS link, loading spinner / indeterminate progress is not announced, display of results does not interrupt URL/feed name read-out

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#603]

**Comment by danielweck** [comment]
Suggestion: ARIA live assertive or polite?

**Comment by danielweck** [comment]
The issue of async UI reactions is more general than this specific OPDS loading spinner problem. Closing this issue as we will need to review / audit other parts of the user interface, hunting for similar lack of responsiveness.
